ioreg/src/builder/setter.rs

```diff
@@ -101,6 +102,27 @@ fn build_new<'a>(cx: &'a ExtCtxt, path: &Vec<String>)
         $setter_ident {
           value: 0,
           mask: 0,
+          write_only: false,
+          reg: reg,
+        }
+      }
+    }
+  ).unwrap())
+}
+
+fn build_new_ignoring_state<'a>(cx: &'a ExtCtxt, path: &Vec<String>)
+                 -> P<ast::ImplItem> {
+  let reg_ty: P<ast::Ty> =
+    cx.ty_ident(DUMMY_SP, utils::path_ident(cx, path));
```

Comment:
to avoid hitting lint errors after #298 is done, should do

```diff
-DUMMY_SP
+reg.name.span
```

Because that's where the name of this register is derived from.